README.md

```diff
@@ -426,14 +426,6 @@ This content lives on https://docs.quantum.ibm.com.
 
 See the section "Syncing content with open source repo" in the internal docs repo's README.
 
-### Deploy tutorials
```

Comment:
You can also remove the h2 header above since there is now only one type of deployment